packages/cli/src/get-filename.ts

```diff
@@ -6,5 +6,5 @@ export const getOutputFilename = () => {
 		console.log('Pass an extra argument <output-filename>.');
 		process.exit(1);
 	}
-	return args._[3];
+	return args._[3].split('.')[0];
```

Comment:
Let's make it more strict so that if a user has an weird configuration like MP4 file but with WebM extension, it doesn't magically replace but instead throws an error.

Let's enforce:

- If user has MP4 format selected, the output extension must be .mp4
- If user has WebM format selected, the output extension must be .webm
- If user has PNG sequence selected, there can not be an extension (if this is hard to do, let's do that it cannot be .mp4 or .webm)

If these conditions are not met, let's `console.log` a message and `process.exit(1)`.

Reply:
I was thinking let user provide only file name. We will added the necessary extension automatically. How's that sounds?

Reply:
Ok, sounds good, if you omit the extension, we will add it automatically.
Let's still allow to explicitly define the extension and validate it, it would be nice that if you specify the exact file it is guaranteed end up being that file. In case you are using it inside a script or something

Reply:
Yeah that sounds like a nice functionality improvement.